bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -2103,7 +2103,7 @@ void BCStateTran::continueSendBatch() {
       if ((rvbGroupDigestsActualSize == 0) || (sb.rvbGroupDigestsExpectedSize != rvbGroupDigestsActualSize)) {
         auto additionalInfo = "Rejecting message - not holding all requested digests (or some other error)" +
```

Comment:
Should getSerializedDigestsOfRvbGroup() be improved to log all possible cases?

Reply:
Sorry, missed this one. Anyway it wouldn't have been fied in this PR.
I will set up a memo on slack to discuss this next week, might create a ticket for it.